src/server/src/routes/student.ts

```diff
@@ -0,0 +1,78 @@
+import StatusCodes from 'http-status-codes';
+import { Request, Response, Router } from 'express';
+import { IStudent } from '@entities/student';
+import { errors } from '@shared/errors';
+import {
+    updateStudent
+} from '@modules/student';
+import logger from '@shared/Logger';
+
+const router = Router();
+
+const { BAD_REQUEST, CREATED, OK, INTERNAL_SERVER_ERROR } = StatusCodes;
+
+interface studentRequest extends Request {
+    body: {
+        student: IStudent;
+    };
+}
+
+/******************************************************************************
+ *   POST Request example - Update - "POST /api/studentProfile/update"
+ ******************************************************************************/
+
+router.post('/update', async (req: studentRequest, res: Response) => {
```

Comment:
/update-profile. Update the one for facultymember to /update-profile too please.

Reply:
Done.